src/Algebra/Graph.hs

```diff
@@ -634,17 +673,20 @@ edgeCountIntR = AIM.edgeCount . toAdjacencyIntMap
 -- @
 vertexList :: Ord a => Graph a -> [a]
 vertexList = Set.toAscList . vertexSet
-{-# INLINE [1] vertexList #-}
+{-# INLINE [2] vertexList #-}
```

Comment:
I think `vertexList` and `edgeList` are good consumers too.

Reply:
Indeed, I even tested it ^^